Nim/Program.cs

```diff
@@ -0,0 +1,48 @@
+﻿Console.WriteLine("Welcome to Nim");
+int matches = 24;
+
+drawAgain:
+Console.WriteLine("Remove 1-3 matches");
+Console.WriteLine("User");
+//int draw = int.Parse(Console.ReadLine());
+if (!int.TryParse(Console.ReadLine(), out int draw))
+{
+    Console.WriteLine("You can't do that, choose 1-3 please");
+    goto drawAgain;
+}
+
+if (draw > 3 ||(draw <=0))
+{
+    Console.WriteLine("Don't do that, choose 1-3 please");
+    goto drawAgain;
+}
+
+matches = matches - draw;
+if (matches > 1)
```

Comment:
You have a bug right now, where the player can win if he draws the last three remaining matches. He's supposed to lose if there's no more matches left. You somehow got that wrong here. Test it!